init/bootstrap.go

```diff
@@ -42,13 +41,6 @@ outer:
 			continue
 		}
 
-		for _, b := range buffer {
```

Comment:
Can't check for 0's, 'cos  the userdata is after the magic string, and its length is unknown